### DISCUSSION TITLE
Fix all statistics issues at once

### DISCUSSION THREAD

**Discussion by tjorim** [frenck/spook#387] [Ideas]
Sometimes someone might have a bunch of statistics issues.
Fixing them one by one can be slow and cumbersome (2 clicks per entity).
Would be nice to have a service that can fix them all at once.
![image](https://github.com/frenck/spook/assets/1294876/6c5d0435-5705-4ec2-9cf1-eec2a270ae5d)

**Comment by tjorim** [comment]
I _think_ this happened after deleting an integration (which was already disabled) but there could be more cases.

**Reply by robchandhok** [reply]
I've seen this when an integration makes some major changes, which usually makes entity names change.   I think this would be a great service - something like "clear all entities with no state errors".  I can see where some stats errors would be hard.

**Comment by wranglatang** [comment]
I would love to see this too!
